### PR TITLE
Support running on standalone Flink cluster

### PR DESCRIPTION
**Pull request by grzegorz8**

### CODE REVIEW

File: docker-compose/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- an "edge" node from which Flink jobs can be scheduled (`flink-sql-runner-node` container).
+- minio instance which serves as AWS S3 substitute.
+- a short-living container `createbuckets` which configures minio, namely, creates necessary buckets, policies and
+  users.
+
+Flink UI can be accessed at `http://localhost:8081`.
+Minio UI can be accessed at `http://localhost:9001` (`user`/`password`).
+
+---
+
```
Comment (andrzejdackiewicz) [inline docker-compose/README.md]: I would add a command for running docker compose to set up the environment before running the example.
`docker compose -f docker-compose/docker-compose.yaml up -d`

**Review by andrzejdackiewicz — APPROVED** [review]
LGTM!

File: docker-compose/README.md (hunk 1 above)
Comment (grzegorz8) [reply docker-compose/README.md]: done

File: deployment-scripts/jobs-deployment/s3.py [hunk 2]
```diff
@@ -51,3 +52,10 @@ def get_latest_object(
             return None
         else:
             return sorted_keys_by_ts_desc[0]
+
+
+def _get_s3_client():
```
Comment (niladrem) [inline deployment-scripts/jobs-deployment/s3.py]: can we use a singleton?

File: python/run-flink-code.py.jinja2 [hunk 3]
```diff
@@ -39,15 +39,23 @@ sql_paths = args.table_definition_path
...
+    table_definitions_params['truststore_password'] = os.environ['KAFKA_CLUSTER_CA_PASSWORD']
+if 'KAFKA_USER_PASSWORD' in os.environ:
+    table_definitions_params['keystore_password'] = os.environ['KAFKA_USER_PASSWORD']
+
 for path in sql_paths:
     if path.startswith('s3://'):
         # Reading table definitions from S3 bucket
         _, path = path.split("://", 1)
-        fs = s3fs.S3FileSystem()
+        if 'AWS_S3_ENDPOINT' in os.environ:
```
Comment (niladrem) [inline python/run-flink-code.py.jinja2]: I think we can change it to inline if

File: docker/docker-compose.yaml [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+version: '3'
+services:
+
+  minio:
+    image: minio/minio
+    environment:
+      - MINIO_ROOT_USER=user
+      - MINIO_ROOT_PASSWORD=password
+    volumes:
+      - /data1/minio:/data
```
Comment (niladrem) [inline docker/docker-compose.yaml]: Pls mention it in the readme

File: deployment-scripts/jobs-deployment/s3.py (hunk 2 above)
Comment (grzegorz8) [reply deployment-scripts/jobs-deployment/s3.py]: done

File: docker/docker-compose.yaml (hunk 4 above)
Comment (grzegorz8) [reply docker/docker-compose.yaml]: I'll remove volumes. For testing purposes minio data does need to be persisted.

File: python/run-flink-code.py.jinja2 (hunk 3 above)
Comment (grzegorz8) [reply python/run-flink-code.py.jinja2]: done

**Review by niladrem — APPROVED** [review]

**Pull request merged**